tx.go

```diff
@@ -1,8 +1,68 @@
 package libra
 
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+
+	"github.com/philippgille/libra-sdk-go/rpc/types"
+)
+
 // Transaction is a transaction of Libra Coins.
 type Transaction struct {
 	RawBytes     []byte
 	SenderPubKey []byte
 	SenderSig    []byte
 }
+
+// TransactionList
+type TransactionList struct {
```

Comment:
An extra type for a transaction list that only contains a slice of transactions is not necessary I think.